bin/create_folders.py

```diff
@@ -143,13 +195,37 @@ def create_folders(
                     access_type=[],
                 )
 
-    # TODO: https://sagebionetworks.jira.com/browse/IBCDPE-809
-    # elif build_or_update == "update":
-    #     update_subfolders(syn, parent_folder_id)
+    elif create_or_update == "update":
+        root_folder_id = syn.findEntityId(name=root_folder_name, parent=project_id)
+
+        file_entity = update_subfolders(
+            syn,
+            predictions_file=predictions_file,
+            submitter_id=submitter_id,
+            parent_id=root_folder_id,
+        )
+        prefix_filename(syn, prefix_name=submission_id, old_file_entity=file_entity)
+
+    else:
+        raise ValueError(
+            "``create_or_update`` must be either 'create' or 'update'. Exiting."
+        )
 
 
 if __name__ == "__main__":
     project_name = sys.argv[1]
     submission_id = sys.argv[2]
     create_or_update = sys.argv[3]
-    create_folders(project_name, submission_id, create_or_update)
+    predictions_file = sys.argv[4] if len(sys.argv) > 4 else None
+
+    if create_or_update == "update" and not predictions_file:
```

Comment:
Generally, I'd recommend that guard statements like this exist within the function you are calling (ie: `create_folders`). In the event `create_folders` gets re-used you won't need to copy this initial guard statement as well.

Reply:
That makes sense to me. Updated